libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -46,6 +46,7 @@ Implemented Papers
 - P2872R3 - Remove ``wstring_convert`` From C++26
 - P3142R0 - Printing Blank Lines with ``println`` (as DR against C++23)
 - P2944R3 - Comparisons for ``reference_wrapper`` (comparison operators for ``reference_wrapper`` only)
+- P2968R2 - Make ``std::ignore`` a first-class object (as DR against C++11)
```

Comment:
This was not voted as DR in the plenary. Why have you implemented it as a DR?

I'm not against changing `operator=(auto&&) const` to `operator=(const auto &) const` for all language versions, but I feel that should be mentioned separately in the release notes.

Reply:
@mordante Thank you for the review. 

Regarding the implementation: I am not sure what makes a paper a DR. https://en.cppreference.com/w/cpp/compiler_support/26 seems to have listed [P2968R2](https://wg21.link/P2968R2) as DR11. I assumed that DR are listed here: https://github.com/cplusplus/draft/issues/7078 but I don't have access to the link.

Why wasn't this voted as DR? To me it would make sense but I can't really tell. Should I restore the original implementation and tests for the pre-C++26 versions? Or keep the current implementation and update release notes as you suggest:

> I'm not against changing operator=(auto&&) const to operator=(const auto &) const for all language versions, but I feel that should be mentioned separately in the release notes.

Reply:
Thanks for explaining why you made this a DR.

Basically something becomes a DR when the committee votes it becomes a DR. The voting of this paper it does not mention a DR. (This information is currently only available for Committee members. For the previous meeting the information is public https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2024/n4980.pdf has the vote "Accept as a Defect Report and apply the changes in P3107R5 (Permit an efficient implementation of std::print) to the C++ working paper.")

So I would not make this paper a DR but keep the new class unconditionally in all language versions. However we should not use `constexpr`, but keep the original macro.

The paper does mention LWG issues, but the paper does not propose itself as a Defect Report.

Reply:
@mordante Thanks. I think I just addressed these comments. Let's see if the CI will get green first.